components/ProductCard/style.js

```diff
@@ -46,6 +46,14 @@ const wishlist = css({
   transform: 'translate3d(0, -50%, 0)',
 }).toString();
 
+const updateImageContainer = css({
+  borderRadius: '0px',
```

Comment:
For pixel values (and some other) you don't need to use a string you can just use the number 
just a hint for the next time ;)

Reply:
(y)